doc/configuring.md

```diff
@@ -363,6 +363,19 @@ genbank_cache: ./genbank_cache
 # the default is set for the all-Genbank database.
 # DEFAULT: 100e9
 prefetch_memory: 100e9
+
+
+### OTHER PARAMETERS
+
+## Error trimming flags
+# fastp_correction: set to ON or 1 for base correction for PE data
+fastp_correction: OFF
```

Comment:
I think we should use YAML booleans - it looks like OFF is a valid one (per https://yaml.org/type/bool.html), but your code below doesn't use boolean value checking.

Reply:
Isn't that checking for boolean?
https://github.com/dib-lab/genome-grist/blob/8f7180f59e07128c8f9c947e9fca1ca9ba2a24d8/genome_grist/conf/Snakefile#L97

Reply:
the 'if' here worries me - if it's a boolean, then you should just be able to say `if config_correction`. Most specifically, '1' should not be magic (which it seems to be here). I don't really know how snakemake's config yaml parser interprets valid vs invalid YAML boolean values, but I would be disturbed to see 'yes' interpreted as false here.

YAML doesn't seem to have a simple answer for what is a valid bool - [this](https://yaml.org/type/bool.html) is a mess! - 
```
 y|Y|yes|Yes|YES|n|N|no|No|NO
|true|True|TRUE|false|False|FALSE
|on|On|ON|off|Off|OFF
```
I guess the simplest thing to do would be to _insist_ that it be a valid bool (True or False), not a '1' or anything else, and complain if it's not.

(I'm disgruntled with YAML because of this kind of thing - Luiz mentioned [an increasing preference for TOML](https://github.com/sourmash-bio/sourmash/issues/1671#issuecomment-989501150), because YAML isn't really that well specified.)